src/Connection.h

```diff
@@ -70,7 +71,8 @@ class Connection : public QObject {
 		/// qmCrypt locks access to csCrypt.
 		QMutex qmCrypt;
 #endif
-		CryptState csCrypt;
+		std::unique_ptr<CryptState> csCryptPtr;
+		CryptState *csCrypt;
```

Comment:
Shouldn't this be deleted in favor of `csCryptPtr`? :thinking:

Reply:
Also, I would simply call it `csCrypt`, there's no need to specify that it's a pointer.